orchestrator/workflow.py

```diff
@@ -323,10 +331,55 @@ def dblogstep(step_: Step, p: Process) -> Process:
         return _exec_steps(step_list, process, dblogstep)
 
     # Make sure we return a form is a sub step has a form
-    form = next((sub_step.form for sub_step in steps if sub_step.form), None)
+    form = next((sub_step.form for sub_step in steps if sub_step.form), None) if extract_form else None
     return make_step_function(func, name, form)
 
 
+def _create_endpoint_step(key: str = DEFAULT_CALLBACK_ROUTE_KEY) -> StepFunc:
+    def stepfunc(process_id: UUID) -> State:
+        token = secrets.token_urlsafe()
+        route = f"/processes/{process_id}/continue/{token}"
```

Comment:
What's the route used for?

Reply:
That's the part of the endpoint used for the callback.
It's a route (or path) and not a full endpoint url, because we are missing the BASE_API_URL, but that can be added in the call to the external system.
It is added to the state so you can access it in the action step. You need to communicate this endpoint to the other system.